docs/user/upgrade.rst

```diff
@@ -341,16 +357,6 @@ requiring changes in user code.
 In addition, the format used to store metadata files has been updated, so older versions of GeoMesa will not be
 able to read metadata created with this version.
 
-Deprecated Modules
```

Comment:
we should probably keep this section. we could maybe update it to 'have been removed' instead of 'will be removed'

Reply:
Sounds good.  I'll do that after we've decided on the language for Spark 2/Scala 2.11 support.